packages/ui/src/theme-web/typography.ts

```diff
@@ -1,7 +1,7 @@
-import { getExtensionTextVariants } from '@leather-wallet/tokens';
+import { getWebTextVariants } from '@leather-wallet/tokens';
```

Comment:
This is going to be a breaking change in the extension so we should release and update it soon